Sources/Views/Cells/AudioMessageCell.swift

```diff
@@ -47,25 +47,51 @@ open class AudioMessageCell: MessageContentCell {
         return durationLabel
     }()
 
+    private lazy var activityIndicatorView: UIActivityIndicatorView = {
```

Comment:
I think we would be ok to keep this public, so others could change the style if they have a darker message bubble color

Reply:
OK.

Reply:
OK.